tests/cart/addItemToCart.spec.js

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@playwright/test';
+const HomePageUrl = 'https://magento.softwaretestingboard.com/';
+
+test('Navigate to top tees', async ({ page }) => {
+  await page.goto(HomePageUrl);
+  await page.locator('#ui-id-5').hover();
+  await page.locator('#ui-id-17').hover();
+  await page.locator('#ui-id-21').click();
+
+  await expect(page.url()).toEqual(
+    'https://magento.softwaretestingboard.com/men/tops-men/tees-men.html'
+  );
+});
+
+test('Filter top tees by color green and select element', async ({ page }) => {
+  await page.goto(
+    'https://magento.softwaretestingboard.com/men/tops-men/tees-men.html'
+  );
+  await page.getByRole('tab', { name: 'Color' }).click();
+  await page.locator('#narrow-by-list [option-id="53"]').click();
+  await expect(page.locator('.product-item')).toHaveCount(4);
+  await page
+    .getByAltText('Atomic Endurance Running Tee (V-neck)-XS-Green')
+    .click();
+
+  await expect(page.url()).toEqual(
+    'https://magento.softwaretestingboard.com/atomic-endurance-running-tee-v-neck.html'
+  );
+});
+
+test('Add item to cart', async ({ page }) => {
+  await page.goto(
+    'https://magento.softwaretestingboard.com/atomic-endurance-running-tee-v-neck.html'
+  );
+  await page.getByLabel('XL').click();
+  await page.getByLabel('Green').click();
+  await page.locator('#qty').fill('3');
```

Comment:
create a reusable method for this part

Reply:
also getByTestId('qty')

embrace the builtin methods, live the builtin methods, breath the builtin methods

Reply:
That is the frst method I tried, but it doesn't find the element